xml/System/Double.xml

```diff
@@ -1815,6 +1815,7 @@
       <Docs>
         <summary>Converts the string representation of a number to its double-precision floating-point number equivalent.</summary>
         <altmember cref="Overload:System.Double.TryParse" />
+        <remarks>In .NET Core 3.0 and later, values that are too large to represent are rounded to <xref:System.Double.PositiveInfinity/> or <xref:System.Double.NegativeInfinity/> as required by the IEEE 754 specification. In prior versions, including .NET Framework, parsing a value that was too large to represent resulted in failure.</remarks>
```

Comment:
I think that xrefs don't work in this context.
```suggestion
        <remarks>In .NET Core 3.0 and later, values that are too large to represent are rounded to <see cref="F:System.Double.PositiveInfinity" /> or <see cref="F:System.Double.NegativeInfinity" /> as required by the IEEE 754 specification. In prior versions, including .NET Framework, parsing a value that was too large to represent resulted in failure.</remarks>
```

Reply:
Ah, thank you - I saw red highlighting in visual studio from `xref` but didn't know the proper alternative - thank you!